src/components/AboutPage.js

```diff
@@ -1,13 +1,88 @@
 import React from 'react';
+import '../styling/about.css';
+
+{/* TODO 
+- scrolling
+- request a project button
+- open projects button
+- style cleanup
+- spacing cleanup
+*/}
 
 const AboutPage = () => (
   <div class="container mt-5">
     <div class="row">
       <div class="col"></div>
       <div class="col-8">
-        <h1>About us</h1>
+        <h1>Build for Black Lives</h1>
+        <p class="p">
+          This is a blurb about who we are and why we started this project and
+          about how it is more about connecting people. Our mission and values 
+          about supporting Black lives. We want allies to educate themselves and
+           use this platform as a way to support Black lives.
+          <br />
+          <br />
+          This is a blurb about who we are and why we started this project and
+          about how it is more about connecting people. Our mission and values 
+          about supporting Black lives. We want allies to educate themselves and
+           use this platform as a way to support Black lives.
+        </p>
       </div>
+    </div>
+
+    <hr />
+    
+    <div class="row">
```

Comment:
in react everything should actually be `className=` and not class sorry this is my bad i didnt change them